CORRELATION_CONTEXT_HTTP_HEADER_FORMAT_RATIONALE.md

```diff
@@ -0,0 +1,32 @@
+# Rationale for Correlation Context HTTP Header Format
+
+This document provides rationale for the decisions made for the `Correlation-Context` header format.
+
+## Purpose of a header
+
+Correlation context header is used to propagate properties not defined in `Trace-Context`. There are two common use cases. First is to define a context on trace initiation. Such context will have customer's identity, high-level operation name and other properties like a flight name. Second use case is to pass the caller's name to the next component. This name-value pair will be overridden by every service to it's own name.
+
+## General considerations
+
+- Should be human readable. Cryptic header will hide the fact of potential information disclosure.
+- Should be appende-able (comma-separated) https://tools.ietf.org/html/rfc7230#page-24 so nodes can add context properties without parsing an existing headers.
+- It is expected that the typical name will be a single word in latin and the value will be a short string in latin or a derivative of an url.
```

Comment:
-1. Counter example: `auth-token={long-base64-string}`

Reply:
url encode will not inflate base64 value too much unless it has a lot of `/` and `+`. It will loose readability in favor of consistent encoding pattern. Do you see this as a major issue?